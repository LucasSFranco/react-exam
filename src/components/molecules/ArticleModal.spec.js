import React from 'react'
import { render } from '@testing-library/react'

import ArticleModal from './ArticleModal'
import articles from '../../../tests/fixtures/articles.json'

jest.mock("../atoms/Modal", () => {
  return {
    Modal: ({ content }) => {
      return <>{content}</>;
    },
  };
});

describe('Article Modal', () => {

  it('properly renders article data', async () => {

    const { container } = render(
      <ArticleModal article={articles[0]}>
        <button />
      </ArticleModal>
    )

    expect(container.firstChild).toMatchSnapshot()
  })

})
