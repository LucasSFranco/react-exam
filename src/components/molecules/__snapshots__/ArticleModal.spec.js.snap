// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article Modal properly renders article data 1`] = `<button />`;
